.github/workflows/integration-test.yml

```diff
@@ -827,7 +797,7 @@ jobs:
     name: 'StopLocalStack'
     runs-on: ubuntu-latest
     if: ${{ always() }}
-    needs: [ StartLocalStack, EC2LinuxIntegrationTest ]
+    needs: [ StartLocalStack, EC2LinuxIntegrationTest, LinuxOnPremIntegrationTest ]
```

Comment:
Does the on-prem test actually use LocalStack? This is specifically for the CA bundle tests.

Reply:
will make this run all tests ec2 runs since I don't see claims anywhere they have different supported feature set.